src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedProjectReference.xaml

```diff
@@ -101,6 +117,21 @@
                     Description="Version of reference.">
     </StringProperty>
 
+    <StringProperty Name="IncludeAssets" 
+                    Visible="True"
+                    DisplayName="IncludeAssets"
+                    Description="Assets to include from this reference" />
+
+    <StringProperty Name="ExcludeAssets" 
```

Comment:
I know you're just applying this consistently with the unresolved version - but all other display names are "Foo Bar", not "FooBar" - can we make the same change there.